lib/sassc/script/value_conversion.rb

```diff
@@ -11,6 +11,12 @@ def self.from_native(native_value, options)
           argument = Script::String.new(value, type)
 
           argument
+        when :sass_number
+          value = Native.number_get_value(native_value)
+          unit = Native.number_get_unit(native_value)
+          argument = Sass::Script::Value::Number.new(value, unit)
```

Comment:
> I need to make this more clear in the code, but we do not need to include `SassC::Script::Number` or `SassC::Script::Map` here *[script.rb]*.
> 
> I have `SassC::Script::String` and `SassC::Script::Color` here because of a mistake that I need to be compatible with.

I removed the additions to script.rb, but I needed to rename Number to make it work:

```diff
- argument = SassC::Script::Number.new(value, unit)
+ argument = Sass::Script::Value::Number.new(value, unit)
```

I'm assuming this is as it should be?